panels/class-debug-bar-deprecated.php

```diff
@@ -91,7 +72,8 @@ function deprecated_function_run( $function, $replacement, $version ) {
 			$message = sprintf( __( '%1$s is <strong>deprecated</strong> since version %2$s with no alternative available.', 'debug-bar' ), $function, $version );
 		}
 
-		$this->deprecated_functions[ $location ] = array( $message, wp_debug_backtrace_summary( null, $bt ) );
+		$key = md5( $location . ':' . $message );
+		$this->deprecated_functions[ $key ] = array( $location, $message, wp_debug_backtrace_summary( null, $bt ) );
```

Comment:
No, as the `$backtrace` variable is already in use in that same function.

`$bt` is used as a pointer to the array entry within a `$backtrace` which is the source of the deprecated function call.
It is used like that in two of the deprecated function related methods.

I agree it would be good to make the variable name more descriptive, but can't think of a good name.

Also, I think that should be a separate PR as that's unrelated to the issue this PR addresses.